pom.xml

```diff
@@ -18,7 +18,7 @@
   <properties>
     <java.level>7</java.level>
     <jackson.version>2.2.3</jackson.version>
-    <jenkins.version>2.7.1</jenkins.version> <!-- Should be kept in sync with Dockerfile FROM statement-->
+    <jenkins.version>2.26-SNAPSHOT</jenkins.version> <!-- Should be kept in sync with Dockerfile FROM statement-->
```

Comment:
We actually won't be able to depend on this it looks like, we need to find a way to make it work with 2.7 lineage as that is the most common one out there and people take a while to upgrade